tests/test_urlgrabber.py

```diff
@@ -27,6 +27,7 @@
 import sabnzbd.urlgrabber as urlgrabber
 import sabnzbd.version
 from sabnzbd.cfg import selftest_host
+from sabnzbd.urlgrabber import filename_from_content_disposition
```

Comment:
No need for this import, already imported a few lines earlier. Simply call the function as `urlgrabber.filename_from_content_disposition()`.